src/sss_client/subid/sss_subid.c

```diff
@@ -18,26 +18,111 @@
 #include <stdlib.h>
 #include <string.h>
 #include <shadow/subid.h>
+#include "sss_cli.h"
 
 
 /* Find all subid ranges delegated to a user.
  *
  * Usage in shadow-utils:
  *      libsubid: get_sub?id_ranges() -> list_owner_ranges()
+ *
+ * SUBID_RANGES Reply:
+ *
+ * 0-3: 32bit unsigned number of UID results
+ * 4-7: 32bit unsigned number of GID results
+ * For each result (sub-uid ranges first):
+ * 0-3: 32bit number with "start" id
+ * 4-7: 32bit number with "count" (range size)
  */
 enum subid_status shadow_subid_list_owner_ranges(const char *user,
```

Comment:
This change seems to belong to the first patch.

Reply:
Actually I kept two patches to easy review a bit, but I was going to squash them after (and add a nice commit message). IMO, having commit with a stub of plugin in the history doesn't have much value.